descriptors/ModuleDescriptor-template.json

```diff
@@ -71,7 +71,11 @@
             "consortia.tenants.item.post"
           ],
           "modulePermissions": [
-            "consortia.consortia-configuration.item.post"
+            "consortia.consortia-configuration.item.post",
+            "perms.users.item.put",
```

Comment:
I believe you need more permissions, this endpoint does get user, post, read permissions, please include permissions based on what endpoint does

Reply:
Added.